apps/munimoney/munimoney-web.yml

```diff
@@ -0,0 +1,73 @@
+---
+- hosts:
+    - munimoney
+  become: yes
+  roles:
+    - dokku_bot.ansible_dokku
+
+  vars:
+    instance_name: "munimoney-{{ env_name }}"
+    app_name: "{{ instance_name }}-web"
+    dokku_daemon_install: false # We install this using a dedicated playbook
+    dokku_plugins:
+      - name: letsencrypt
+        url: https://github.com/dokku/dokku-letsencrypt.git
+
+  tasks:
+    - name: "Dokku app exists"
+      dokku_app:
+        app: "{{ app_name }}"
+
+    - name: "Dokku app config is set"
+      vars:
+        postgres_password: "{{ lookup('passwordstore', 'apps/munimoney/{{ env_name }}/POSTGRES')}}"
+        postgres_hostname: "{{ lookup('passwordstore', 'apps/munimoney/{{ env_name }}/POSTGRES subkey=hostname')}}"
+        postgres_username: "{{ lookup('passwordstore', 'apps/munimoney/{{ env_name }}/POSTGRES subkey=username')}}"
+        postgres_database: "{{ lookup('passwordstore', 'apps/munimoney/{{ env_name }}/POSTGRES subkey=database')}}"
+      dokku_config:
+        app: "{{ app_name }}"
+        config:
+          AWS_ACCESS_KEY_ID: "{{ lookup('passwordstore', 'apps/munimoney/{{ env_name }}/AWS subkey=ACCESS_KEY_ID')}}"
+          AWS_SECRET_ACCESS_KEY: "{{ lookup('passwordstore', 'apps/munimoney/{{ env_name }}/AWS')}}"
+          AWS_STORAGE_BUCKET_NAME: "munimoney-media"
+          DATABASE_URL: "postgres://{{ postgres_username }}:{{ postgres_password }}@{{ postgres_hostname }}:5432/{{ postgres_database }}"
+          DJANGO_DEBUG: "true"
```

Comment:
let's make this false - it's publicly-accessible and I think someone forgot it in true while trying something